cli/running/watchdog.go

```diff
@@ -10,6 +10,19 @@ import (
 	"github.com/tarantool/tt/cli/ttlog"
 )
 
+// Provider interface provides Watchdog methods to get objects whose creation
+// and updating may depend on changing external parameters (such as configuration
+// file).
+type Provider interface {
```

Comment:
I don't have an idea for a good name yet, but "Provider" looks too common for me. and does not give any clue of what it is for.

Reply:
It is a facade for interaction with the outside world. I'm willing to settle for another name, but I couldn't think of a good enough one.